sympy/core/relational.py

```diff
@@ -109,13 +119,16 @@ def canonical(self):
         """
         r = self
         if r.func in (Ge, Gt):
-            r = r.reversed
+            if _coeff_isneg(r.rhs):
```

Comment:
I feel that Gt should be converted to Lt only under more special conditions. So I would personally prefer conditions like `x > a + b`, `x > f(a)` and even `x > a` when `x` is variable and `a` is constant. I think that it is more informative to have the variable on the left hand side. (That is the order typically used in verbal communication: "We have `<some property or expression of x>`when x is greater/smaller than `<something>`".)

Reply:
It's complicated, because something like 1 < x < 4, which has to be printed (at least with the str printer) using &&, is easier read as (1 < x) && (x < 4) than (x > 1) && (x < 4).  

My take here would be to put effort in the pretty printers (`pprint` and `latex`) to print things better, and not worry as much about the expression form itself.